pkg/api/plugin.go

```diff
@@ -11,10 +11,11 @@ import (
 type ContextKey string
 
 const (
-	ContextKeyClientID     ContextKey = "ClientID"
-	ContextKeyClientSecret ContextKey = "ClientSecret"
-	ContextKeyTenantID     ContextKey = "TenantID"
-	ContextAcceptLanguages ContextKey = "AcceptLanguages"
+	ContextKeyClientID         ContextKey = "ClientID"
```

Comment:
I would name these differently now e.g. `ContextKeyCloudProviderClientID` to distinguish its purpose from `ContextKeyClientAuthorizer`. 

In fact now that plugin is not going to use cloud provider creds for deploying resource is there a need to continue keeping these in the context instead of just in `OpenShiftManagedCluster.Properties.ServicePrincipalProfile`?

Reply:
Oops, wrong button, meant to submit a comment.  Retyping : 

Possibly, only the line : ContextKeyClientAuthorizer ContextKey = "ClientAuthorizer" has been added by this commit. The other constants were just re-indented by "go fmt". Nevertheless, I will revisit this space and cleanup accordingly -- renaming and removing/moving keys that aren't used or belong only in the fakerp pkg.